internal/core/Cargo.toml

```diff
@@ -12,7 +12,7 @@ repository = "https://github.com/slint-ui/slint"
 homepage = "https://slint-ui.com"
 categories = ["gui", "development-tools"]
 keywords = ["gui", "ui", "toolkit", "graphics", "design"]
-rust-version = "1.60"
+rust-version = "1.62"
```

Comment:
do we have a test for this?

Reply:
No, the CI only tests with the "global" MSRV 1.64.